.travis.yml

```diff
@@ -2,8 +2,9 @@ os:
   - linux
 language: node_js
 node_js:
-  - "6.9"
-  - "4.6"
+  - 4
+  - 5
```

Comment:
If we want to only support [Node.js LTS versions](https://github.com/nodejs/LTS#lts-schedule), then we should drop testing against Node v5.